terraform/app.tf

```diff
@@ -97,11 +97,25 @@ resource "azurerm_mssql_server" "mssql_server" {
   administrator_login_password = local.infrastructure_secrets.SQL_ADMIN_PASSWORD
 }
 
-#MSSQL Database
 resource "azurerm_mssql_database" "mssql_database" {
   name        = local.mssql_database
   server_id   = azurerm_mssql_server.mssql_server.id
   collation   = "SQL_Latin1_General_CP1_CI_AS"
   sku_name    = local.mssql_sku_name
   max_size_gb = local.mssql_max_size_gb
 }
+
+resource "azurerm_application_insights" "web_app_insights" {
```

Comment:
Can `web_app_insights` be changed to `function_app_insights` so it's aligns with the resource it's being used by

Reply:
fixed